worker/src/tests/mod.rs

```diff
@@ -39,6 +42,22 @@ pub fn run_enclave_tests(matches: &ArgMatches, port: &str) {
     crate::db_handler::DBHandler::send_data_to_enclave(eid)
         .expect("Failed to send data to enclave");
 
+    let mut api = Api::new(String::from("ws://127.0.0.1:9994"))
+        .unwrap()
+        .set_signer(AccountKeyring::Alice.pair());
+    let genesis_hash = api.genesis_hash.as_bytes().to_vec();
+
+    let tee_accountid = enclave_account(eid);
+
+    // start disk & ipfs snapshotting
+    crate::polkadex_db::start_snapshot_loop(
```

Comment:
Is that neccessary for unti testing in enclave?

Reply:
The more we need to add here, the less "unit" testing it will be. With emphasis on unti.

Reply:
No, thats just so i can test it easily while developing it